### PR TITLE
RHMAP-11027 Update README to remove Cordova comments

### PR DESCRIPTION
**Pull request by aidenkeating**
Following up from this comment: 

```
Readme documentation claims that fh-advanced-webapp-blank-app is actually an Cordova application.

This is not really true, it is an advanced web application. 

Readme should be updated to reflect that.
```

### CODE REVIEW

**Comment by aidenkeating** [comment]
@danielpassos Would you mind taking a quick look at this? Can you see anything else major to change?

File: README.md [hunk 1]
```diff
@@ -2,20 +2,20 @@ fh-advanced-webapp-blank-app
...
 Level: Intermediate   
-Technologies: Javascript, Cordova, RHMAP   
-Summary: An Hybrid client side template with RHMAP.   
-Community Project: [Feed Henry](http://feedhenry.org)   
+Technologies: Javascript, RHMAP   
+Summary: An advanced web application template to use with RHMAP.   
+Community Project: [FeedHenry](http://feedhenry.org)   
 Target Product: RHMAP   
 Product Versions: RHMAP 3.8.0+   
 Source: https://github.com/feedhenry-templates/sync-cordova-app
```
Comment (jcesarmobile) [inline README.md]: I think the source should be this repo url, not sync-cordova-app url

File: README.md [hunk 2]
```diff
@@ -2,20 +2,20 @@ fh-advanced-webapp-blank-app
...
+Technologies: Javascript, RHMAP   
+Summary: An advanced web application template to use with RHMAP.   
+Community Project: [FeedHenry](http://feedhenry.org)   
 Target Product: RHMAP   
 Product Versions: RHMAP 3.8.0+   
 Source: https://github.com/feedhenry-templates/sync-cordova-app   
-Prerequisites: fh-js-sdk : 2.14.+, Cordova 5.0+   
+Prerequisites: fh-js-sdk : 2.14.+ 
 
 > Note: since RHMAP 3.14, this repository is deprecated in favor of [blank-cordova-app](https://github.com/feedhenry-templates/blank-cordova-app)
```
Comment (jcesarmobile) [inline README.md]: We should remove the deprecation note, we removed the template, but had to bring it back and we will continue maintaining it.

File: README.md (hunk 1 above)
Comment (aidenkeating) [inline README.md]: I don't know how I missed that, thanks 👍

**Pull request merged**